cookie-banner-rules-list.json

```diff
@@ -5240,6 +5240,30 @@
       "cookies": {},
       "id": "8308357f-6a66-433d-bfc3-de401410c350",
       "domains": ["hubspot.com"]
+    },
+    {
+      "id": "63104024-41b5-4be5-8019-2c59eaaf612c",
+      "domains": ["steamcommunity.com", "steampowered.com"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "cookieSettings",
+            "value": "%7B%22version%22%3A1%2C%22preference_state%22%3A1%2C%22content_customization%22%3Anull%2C%22valve_analytics%22%3Anull%2C%22third_party_analytics%22%3Anull%2C%22third_party_content%22%3Anull%2C%22utm_enabled%22%3Atrue%7D"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "cookieSettings",
+            "value": "%7B%22version%22%3A1%2C%22preference_state%22%3A2%2C%22content_customization%22%3Anull%2C%22valve_analytics%22%3Anull%2C%22third_party_analytics%22%3Anull%2C%22third_party_content%22%3Anull%2C%22utm_enabled%22%3Atrue%7D",
+            "isSecure": true
+          }
+        ]
+      },
+      "click": {
```

Comment:
The click rule is correct but Firefox still fails to handle the banner because it appears after the timeout of 3 seconds. If we don't have a way to work around this (I can't think of something off-hand) we should consider dropping this part of the rule and solely rely on the injected cookie (which works well!).

Reply:
You can test only the click rule if you set the pref `cookiebanners.cookieInjector.enabled` to `false`.

Reply:
I am aware of this minor inconvenience and have already written about it in the second paragraph of the description of this pull request.

I can also add that on steamcommunity.com, there is no such delay before this cookie banner appears as there is, for example, on store.steampowered.com, so this click rule serves its purpose (at least partially).

Additionally, I don't think it is a good idea to create completely separate rules for the steamcommunity.com and steampowered.com domains, as they share the same cookie injector rule